### ISSUE TITLE
Add 32 bit windows qt exe

### ISSUE THREAD

**Issue by NullFunctor** [NullFunctor/iBTC#7]
Add 32 bit windows qt exe

**Comment by AlphaSerpentis** [comment]
That shouldn't be much of an issue, all you need to do is have the necessary dependencies and compile it for a 32-bit Windows system.

https://github.com/NullFunctor/iBTC/blob/develop/doc/build-windows.md#building-for-32-bit-windows

**Comment by pengailbitco** [comment]
So everybody can make windows wallet now..perfect

**Comment by AlphaSerpentis** [comment]
Yeah, you could build any particular version really as long as you have the time, the dependencies, and the proper configuration.

I used to build Windows 32-bit/64-bit on a Linux ARM system for other projects in the past.

**Comment by a1love** [comment]
c:\workspace\bitcoin

**Comment by a1love** [comment]
sudo apt install g++-mingw-w64-
